FinniversKit/Sources/Components/Button/IconButton/IconButton.swift

```diff
@@ -32,6 +32,7 @@ public final class IconButton: UIButton {
     public init(style: Style, withAutoLayout: Bool = false) {
         self.style = style
         super.init(frame: .zero)
+        accessibilityTraits = .button
```

Comment:
Why is this needed when it's subclassing `UIButton`?

Reply:
Indeed, we don't need it here 👍